gapii/cc/spy_base.h

```diff
@@ -48,6 +48,7 @@
 #include <unordered_map>
 
 namespace gapii {
+const uint8_t kAllAPIs = 0xFF;
```

Comment:
Probably we can change the `getEncoder` to make is accept a bit mask, instead the shift value, so we can avoid using this 'mask' like thing with the 'shift value' like thing in should_trace()? Hmm, probably not in this CL anyway.

Reply:
Yeah, I would prefer to leave this for a future CL.